assets/operatingInfo/za.private.json

```diff
@@ -0,0 +1,15 @@
+{
+  "timezone": "Africa/Johannesburg",
+  "holidays": { 
+    "12/25/2021": "Christmas"
+  },
+  "operatingHours": {
+    "1": [{ "open": 9, "close": 11 }, { "open": 13, "close": 18 }],
+    "2": [{ "open": 9, "close": 11 }, { "open": 13, "close": 18 }],
+    "3": [{ "open": 9, "close": 11 }, { "open": 13, "close": 18 }],
+    "4": [{ "open": 9, "close": 11 }, { "open": 13, "close": 18 }],
+    "5": [{ "open": 9, "close": 11 }, { "open": 13, "close": 18 }],
+    "6": [],
+    "7": []
```

Comment:
Empty array means there are no open shifts for this day. This is helpful in the below expression of the function
`const isOpenNow = operatingHours[dayOfWeek].some(isInOpenShift);`
as an empty array will return `false` for any callback provided to `some` method.